gke/create_ip.sh

```diff
@@ -19,9 +19,9 @@ PROJECT_ID=$(yq eval '.project' config.yaml)
 gcloud config set project $PROJECT_ID
 
 # Create a static IP address for the API
-gcloud compute addresses create website-ip --global
```

Comment:
What does this change do? I'm still seeing some references to `website-ip` floating around ( in `update_config_cluster.sh` and `worldbank.yaml`) - is this OK?

Reply:
This change is trying to match the ip created in this script with the ip helm is expecting here: https://github.com/datacommonsorg/website/blob/84eea6a8a5521e7eede559835d8a06c3ddab85c2/deploy/helm_charts/dc_website/values.yaml#L45

Thanks for the catch! Also updated `update_config_cluster.sh`. The worldbank project still uses `website-ip`, so leaving that as is.